CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Changes
 * [Developer]: Removed deprecated Long constructor.
 * [Developer]: Updated `webpack-dev-server` to fix security vulnerability caused by `websocket-extensions` 0.1.3.
 * [UI]: Update project metadata templates listing page to use Ant Design.
+* [UI]: Allowing users to synchronize assembly data from remote APIs.
```

Comment:
Change this to `Added functionality for users to synchronize assembly data from remote APIs.`

Reply:
Updated in 595a6ad419